BrowserSelect/Browser.cs

```diff
@@ -127,30 +127,12 @@ public static List<Browser> find(bool update = false)
                 //remove duplicates
                 browsers = browsers.GroupBy(browser => browser.exec)
                     .Select(group => group.First()).ToList();
-                //Check for Chrome Profiles
-                Browser BrowserChrome = browsers.FirstOrDefault(x => x.name == "Google Chrome");
-                if (BrowserChrome != null)
-                {
-                    string ChromeUserDataDir = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), @"Google\Chrome\User Data");
-                    List<string> ChromeProfiles = FindChromeProfiles(ChromeUserDataDir);
 
-                    if (ChromeProfiles.Count > 1)
-                    {
-                        //add the Chrome instances and remove the default one
-                        foreach (string Profile in ChromeProfiles)
-                        {
-                            browsers.Add(new Browser()
-                            {
-                                name = "Chrome (" + GetChromeProfileName(ChromeUserDataDir + "\\" + Profile) + ")",
-                                exec = BrowserChrome.exec,
-                                icon = icon2String( IconExtractor.fromFile(ChromeUserDataDir + "\\" + Profile + "\\Google Profile.ico") ),
-                                additionalArgs = String.Format("--profile-directory={0}", Profile)
-                            });
-                        }
-                        browsers.Remove(BrowserChrome);
-                        browsers = browsers.OrderBy(x => x.name).ToList();
-                    }
-                }
+                //check for edge chromium profiles
+                AddChromeProfiles(browsers, "Microsoft Edge", @"Microsoft\Edge\User Data", "Edge Profile.ico");
```

Comment:
These seem to be duplicated changes from your previous pull request. Did you perhaps forget to sync with the master branch before creating this pull request? (On your forked repo it says `This branch is 2 commits ahead, 1 commit behind zumoshi:master.`, it shouldn't be behind when you create a pull request)